cmake/KokkosKernels_config.h.in

```diff
@@ -119,6 +119,13 @@
 #endif
 #endif
 
+#if !defined(KOKKOS_ENABLE_CUDA) \
+  && !defined(KOKKOS_ENABLE_HIP)/* \
+  && !defined(KOKKOS_ENABLE_SYCL) \
```

Comment:
Wouldn't it be correct to not comment out SYCL/OpenMPTarget here?

Reply:
Yes, I actually should have them even though the OpenMPTarget backend is not merged yet, the Kokkos end of it already exist so yes I can enable that directly, let me make that change...